generators/server/templates/gradle/docker.gradle.ejs

```diff
@@ -27,6 +27,11 @@ buildscript {
 
 apply plugin: com.google.cloud.tools.jib.gradle.JibPlugin
 
+task copyWwwIntoStatic (type: Copy) {
```

Comment:
I think this can be guarded by `<%_ if (!skipClient) { _%>` as we don't need to do that when there is no client.
Also it should be `'<%= CLIENT_DIST_DIR %>'` instead of hard coded `build/www` I think.